### DISCUSSION TITLE
Custom Dash Controls

### DISCUSSION THREAD

**Discussion by ljilekor** [idling-mind/flowfunc#15] [Q&A]
Is it possible to implement Dash as Custom Port Controls?

If yes, how can one achieve this?

**Comment by vmmvp90** [comment]
Good

**Comment by idling-mind** [comment]
It's defenitely something I wanted to implement. However, I have not figured out how to achieve this.
One workaround is to render controls outside of flowfunc by tracking which node was double clicked. (simliar to how [Node-Red](https://nodered.org/) does it). To get the double clicked node, use the property `double_clicked_node` of the `Flowfunc` object.
You'll also have to manage the state of each node's control's data outside of flowfunc as well. (may be in a `dcc.Store`)
